tests/Doctrine/Tests/ORM/Query/SelectSqlGenerationTest.php

```diff
@@ -289,6 +290,22 @@ public function testSupportsOrderByDesc()
         );
     }
 
+    public function testSupportsOrderByStartingWithFunction()
+    {
+        $this->assertSqlGeneration(
+            'SELECT u FROM Doctrine\Tests\Models\Forum\ForumUser u ORDER BY LENGTH(u.username) asc',
+            'SELECT f0_.id AS id_0, f0_.username AS username_1 FROM forum_users f0_ ORDER BY LENGTH(f0_.username) ASC'
+        );
+    }
```

Comment:
This test passes already without a need for your patch, did you add it to make sure you were not breaking that feature? Might warrant a separate commit too.

Reply:
Yes, it's a complementary test that in fact is useless because it is not even impacted by the change (there is NO confusion possible with tsuch order by tokens), so I'll remove it.